webapp/channels/src/packages/mattermost-redux/src/reducers/entities/posts.ts

```diff
@@ -1575,6 +1575,20 @@ export function limitedViews(
     }
 }
 
+export interface ErrorsReducer {
+    postEditHistory?: true;
+}
+const emptyErrors = {};
+export function errors(state: ErrorsReducer = emptyErrors, action: GenericAction) {
```

Comment:
nit: please change the name to something more explicit so it doesn't conflict with the state property. I suggest something like `fetchErrors`:)

Reply:
Also done!